Forge.TreeWalker.UnitTests/test/TreeWalkerUnitTestsWithV1Callbacks.cs

```diff
@@ -0,0 +1,1143 @@
+//-----------------------------------------------------------------------
+// <copyright file="TreeWalkerUnitTestsWithV1Callbacks.cs" company="Microsoft">
```

Comment:
>TreeWalkerUnitTestsWithV1Callbacks [](start = 20, length = 34)

We should be able to test the new behavior in a few tests without having to duplicate the entire test file for this one change.

Reply:
yes, we should be able to update a few tests. My thought is to use a copied file to cover all existing tests with V1, so V1 is fully verified. Then update the existing tests with V2, so V2 are fully covered. And we can delete "TreeWalkerUnitTestsWithV1Callbacks.cs" in future when we officially retire V1 interface.

---
In reply to: [504215906](https://github.com/microsoft/Forge/pull/32#discussion_r504215906) [](ancestors = 504215906)